Packs/ThycoticDSV/Integrations/ThycoticDSV/README.md

```diff
@@ -0,0 +1,73 @@
+Thycotic DevOps Secrets Vault is a high velocity vault that centralizes secrets management, enforces access, and provides automated logging trails. This cloud-based solution is platform agnostic and designed to replace hard-coded credentials in applications, micro-services, DevOps tools, and robotic process automation. This vault ensures IT, DevOps and Security teams the speed and agility needed to stay competitive without sacrificing security.
```

Comment:
This text seems to be present 3 times, which is unnecessary

Reply:
@guykeller 
Differend :-) Fixed